README.md

```diff
@@ -1,307 +1,51 @@
-Make sure you have the proper permission to the registry if the above commands don’t work.
-
-**Install the CRDs into the cluster:**
-
-```sh
-make install
-```
-
-**Deploy the Manager to the cluster with the image specified by `IMG`:**
-
-```sh
-export NAC_TAG="v0.0.1"
-export IMG_REGISTRY="quay.io/<USER>/oadp-nac"
-
-make deploy IMG="${IMG_REGISTRY}:${NAC_REV}"
-```
-
-**Check your deployment**
-
-```sh
-oc get all -n oadp-nac-system
-```
-
-Result:
-```sh
-NAME                                               READY   STATUS    RESTARTS   AGE
-pod/oadp-nac-controller-manager-74bbf4577b-nssw4   2/2     Running   0          3m7s
-
-NAME                                                  TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
-service/oadp-nac-controller-manager-metrics-service   ClusterIP   172.30.201.185   <none>        8443/TCP   3m8s
-
-NAME                                          READY   UP-TO-DATE   AVAILABLE   AGE
-deployment.apps/oadp-nac-controller-manager   1/1     1            1           3m7s
-
-NAME                                                     DESIRED   CURRENT   READY   AGE
-replicaset.apps/oadp-nac-controller-manager-74bbf4577b   1         1         1       3m7s
-
-```
-
-> **NOTE**: If you encounter RBAC errors, you may need to grant yourself cluster-admin
-privileges or be logged in as admin.
+Non Admin Controller
 
-**Create instances of your solution**
-You can apply the samples (examples) from the config/sample:
+[![Continuos Integration](https://github.com/migtools/oadp-non-admin/actions/workflows/ci.yml/badge.svg)](https://github.com/migtools/oadp-non-admin/actions/workflows/ci.yml)
+[![OADP version compatibility check](https://github.com/migtools/oadp-non-admin/actions/workflows/oadp-compatibility-check.yml/badge.svg)](https://github.com/migtools/oadp-non-admin/actions/workflows/oadp-compatibility-check.yml)
 
-```sh
-kubectl apply -k config/samples/
-```
+<!-- TODO add Official documentation link once it is created -->
 
->**NOTE**: Ensure that the samples has default values to test it out.
+Documentation in this repository are considered unofficial and for development purposes only.
 
-### To Uninstall
-**Delete the instances (CRs) from the cluster:**
-
-```sh
-kubectl delete -k config/samples/
-```
-
-**Delete the APIs(CRDs) from the cluster:**
-
-```sh
-make uninstall
-```
-
-**UnDeploy the controller from the cluster:**
-
-```sh
-make undeploy
-```
-
-## Project Distribution
-
-Following are the steps to build the installer and distribute this project to users.
+## Description
 
-1. Build the installer for the image built and published in the registry:
+This open source controller adds the non admin feature to [OADP operator](https://github.com/openshift/oadp-operator). With it, cluster admins can configure which namespaces non admin users can backup/restore.
 
-```sh
-make build-installer IMG=<some-registry>/oadp-nac:tag
-```
+## Getting Started
```

Comment:
Add a sample application install/usage to test backup/restore

Use this app https://github.com/migtools/oadp-non-admin/pull/34